website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+
+<!--truncate-->
+
+## Improved Kubernetes Workflow
+Podman Desktop facilitates the application development process right from containerization to Kubernetes deployment. With a focus on improving the Kubernetes workflow, we’ve introduced a set of new features.
+
+**1. Custom Kubeconfig file path**
+
+Starting v0.10, users can provide their kubeconfig path location instead of letting Podman Desktop use the default path i.e. `$HOME/.kube/config`. This can be done by going to `Settings > Preferences > Kubernetes: Kubeconfig`
+
+![custom kubeconfig file path](img/podman-desktop-release-0.10/custom-kubeconfig.png)
+
+**2. Play Kubernetes YAML from the Containers list**
+
+Development workflows can require developers to leverage existing Kubernetes YAML files. Podman Desktop now supports playing Pods or Containers from an existing Kubernetes YAML File. 
+
+This can be done in the ‘Containers’ or 'Pods' screen of the application by clicking the `Play Kubernetes YAML` button and playing the existing YAML file.
+
+**3. Kube context automatically reloaded when updated**
+
+One can now reload kube context when file is changed or if content is changing.
+
+Launch Podman Desktop and then try to change the current kube context either by using the tray icon or with `kubectl` tool and then try to use ‘Deploy to Kubernetes’ feature, the selected context should be displayed.
+
+## UI Improvements
+
+**1. Kebab menu**
+
+With the latest release, you will see significant changes with respect to the options and how they are displayed for images and containers. Previously, there was an array of buttons over each container or image component which has now been replaced with a ‘kebab’ menu. This is more user-friendly and keeps all options organized within a single component. By clicking the three dot button, you can open the menu and choose the option depending on your workflow.
+
+![kebab menu](img/podman-desktop-release-0.10/kebab-menu.png)
+
+**2. Containers**
+
+The list of containers now also displays the age of the container as well as the image used to start the container in separate columns alongside the container name.
+
+**3. Responsive Design**
+
+On account of the effort to improve the User Experience, we have made the UI more responsive to window size changes. This includes animations as well as consistent design across all sections of the application.
+
+**4. Registries**
+
+We have also updated the UI of the page where we add/update registries in ‘Settings’.
+
+![registries](img/podman-desktop-release-0.10/registries.png)
+
+**5. Font and text**
+
+Other improvements include making font colors across the application consistent. Significant efforts have been made to display information correctly whether they are container logs or other text information.
```

Comment:
I don't understand what it means

Reply:
Referring to these PRs - 
https://github.com/containers/podman-desktop/pull/802
https://github.com/containers/podman-desktop/pull/904

Reply:
so, here, it's the same comment than before.
Why not saying: 'Increase the text contrast in the images list' and 'use the full height in the logs screen of the container details'

because to me `Significant efforts have been made to display information correctly whether they are` is not precise.

Reply:
https://github.com/containers/podman-desktop/pull/943/commits/abc3da65ea3d6a7da8159e42e4f07fd424eb1655